recipe/meta.yaml

```diff
@@ -1,35 +1,35 @@
 {% set name = "pluggy" %}
 {% set version = "1.0.0" %}
-{% set sha256 = "4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name [0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pluggy-{{ version }}.tar.gz
+  sha256: 4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159
 
 build:
-  number: 4
-  skip: true  # [py<35]
-  script: python -m pip install . --no-deps --ignore-installed
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 5
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.6
+    - setuptools-scm
     - pip
-    - setuptools_scm
   run:
-    - python
-    - importlib_metadata >=0.12  # [py<38]
+    - python >=3.6
+    - importlib-metadata >=0.12
```

Comment:
```suggestion
```

Reply:
We declare it to be safe on all versions; is that not correct? https://conda-forge.org/docs/maintainer/knowledge_base.html#non-version-specific-python-packages

Reply:
Yes, but seeing as the non `noarch` packages will support only `py>=38`, it's fine to impose that condition on the `noarch` package as well.